### PR TITLE
[12.0] [ADD] new module account_voucher_print

### PR DESCRIPTION
**Pull request by primes2h**
This module allows printing of sale and purchase receipts.

### CODE REVIEW

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Review by marcelsavegnago — APPROVED** [review]

**Review by marcelofrare — APPROVED** [review]
functional LGTM

File: account_voucher_print/views/report_receipt.xml [hunk 1]
```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (sergiocorato) [inline account_voucher_print/views/report_receipt.xml]: `<data>` is no more needed

**Review by sergiocorato — CHANGES_REQUESTED** [review]

File: account_voucher_print/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    'author': 'Pordenone Linux User Group (PNLUG), Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/account-financial-reporting',
+    'license': 'AGPL-3',
+    'depends': [
+        'account_voucher'],
+    'data': ['views/report_receipt.xml',
+             'views/account_report.xml'],
+    'installable': True,
+    'application': False,
+    'auto_install': False,
```
Comment (sergiocorato) [inline account_voucher_print/__manifest__.py]:
```suggestion
```

File: account_voucher_print/views/account_report.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (sergiocorato) [inline account_voucher_print/views/account_report.xml]: other unuseful `<data>`

**Review by sergiocorato — CHANGES_REQUESTED** [review]

File: account_voucher_print/views/report_receipt.xml (hunk 1 above)
Comment (primes2h) [reply account_voucher_print/views/report_receipt.xml]: Removed, thanks!

File: account_voucher_print/views/account_report.xml (hunk 3 above)
Comment (primes2h) [reply account_voucher_print/views/account_report.xml]: Removed as well, thanks!

File: account_voucher_print/__manifest__.py (hunk 2 above)
Comment (primes2h) [reply account_voucher_print/__manifest__.py]: Done.

**Review by sergiocorato — APPROVED** [review]

**Pull request merged**